services/ui-src/src/types/states.ts

```diff
@@ -1,6 +1,25 @@
 import { ParentPageTemplate, PageData, Report } from "types/report";
 import React from "react";
-import { HcbsUser } from "types";
+import { AdminBannerData, ErrorVerbiage, HcbsUser } from "types";
+
+// initial admin banner state
+export interface AdminBannerState {
+  // INITIAL STATE
```

Comment:
I understand that these comments are copied from elsewhere, but they are wrong. While an interface can be used to represent state, it cannot contain state, initial or otherwise.

The initial state is actually defined in state/useStore.ts, lines 32-57.

```suggestion
export interface AdminBannerState {
```